src/core/connectors.js

```diff
@@ -1646,6 +1646,11 @@ const connectors = [{
 	],
 	js: 'connectors/idagio.js',
 	id: 'idagio',
+}, {
+	label: 'Relaxfm',
```

Comment:
Could you also update the label to "Relax FM", as it's displayed on the website?

Reply:
Updated the label.